### DISCUSSION TITLE
weboberfläche

### DISCUSSION THREAD

**Discussion by schneefuch8** [ciur/papermerge#512] [Q&A]
komme nicht auf die weboberfäche habe versucht den port zu ändern da ich den port 8000 und den port 80 schon benutze und benutze auch schon nginx

**Comment by ProEsizal** [comment]
du kannst doch den Port einfach ändern und dann speichern und dann gibst du in deinem Browser die ip ein und am enden den port auf den du papermerge geändert hast
